werewolves/lib/ui/pages/start/main_menu.dart

```diff
@@ -17,9 +14,10 @@ class StartPage extends StatelessWidget {
         body: Center(
             child: Column(children: [
           SizedBox(height: distanceBetweenAppBarAndImage),
-          Image.asset('assets/images/werewolves_online.png'), //TODO REPLACE WITH OUR OWN ART
+          Image.asset(
+              'assets/images/werewolves_online.png'), //TODO REPLACE WITH OUR OWN ART
```

Comment:
Suggestion: if you add a `,` after the string the autoformat creates a better looking line
```
Image.asset(
  'assets/images/werewolves_online.png',
),
```

Most importantly, to avoid losing the todos in the code forever, I suggest you [create an issue](https://github.com/werewolves-online/werewolves/issues/new) and add a link to it with the TODO comment